src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -14,14 +20,36 @@
 
     public static final String COMMAND_WORD = "find";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all assignments whose names contain any of "
-            + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
-            + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
-            + "Example: " + COMMAND_WORD + " alice bob charlie";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all assignments"
+            + " by its NAME, MODULE CODE, DEADLINE or PRIORITY. "
+            + "Finding is done one field at a time.\n"
+            + "DEADLINE keywords are in the format dd-MM-yyyy or HHmm, which allows finding of assignments by time "
+            + "and date separately.\n"
+            + "Parameters:" + PREFIX_NAME + "NAME [MORE NAMES] or\n"
```

Comment:
Slight nitpick but since you gave the date format before the time format, you could consider using "... finding of assignments by **date and time** separately", instead of "... finding of assignments by **time and date** separately". But it is fine if you leave it as it is too :)

Reply:
Amended!